wiki/Help_Centre/Account/en.md

```diff
@@ -0,0 +1,212 @@
+# Account
```

Comment:
iirc ephemeral wanted the articles to have lots of tags so that they could be easily searchable

Reply:
+1